.scala-steward.conf

```diff
@@ -0,0 +1 @@
+updates.ignore = [ { groupId = "org.slf4j", artifactId = "slf4j-simple" } ]
```

Comment:
sneaking this into this PR as I can't be bothered to make a separate branch, commit and PR for it